### PR TITLE
K8SPXC-986-cleaning-users.go:

### PR DESCRIPTION
**Pull request by theTibi**
[K8SPXC-986](https://jira.percona.com/browse/K8SPXC-986) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

* MySQL can not rollback create user or grants of flush privileges.
* Removing the unnecessary code to make it cleaner

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/haproxy.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/cross-site.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/scheduled-backup.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/upgrade-haproxy.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/recreate.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/big-data.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/storage.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/scaling-proxysql.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/self-healing.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/self-healing-chaos.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/init-deploy.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/restore-to-encrypted-cluster.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/security-context.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/self-healing-advanced.log) |
| self-healing-advanced-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/self-healing-advanced-chaos.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/upgrade-proxysql.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/limits.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/demand-backup.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/monitoring-2-0.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/operator-self-healing-chaos.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/smart-update.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/affinity.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/one-pod.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/auto-tuning.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/pitr.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/users.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/upgrade-consistency.log) |
| demand-backup-encrypted-with-tls | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/demand-backup-encrypted-with-tls.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/tls-issue-self.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/tls-issue-cert-manager.log) |
| tls-issue-cert-manager-ref | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/tls-issue-cert-manager-ref.log) |
| validation-hook | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/4976aec2/validation-hook.log) |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1115/commits/4976aec2fd76c6313e3cca0a4b5459f32a746e28
image: `perconalab/percona-xtradb-cluster-operator:PR-1115-4976aec2`

File: pkg/pxc/users/users.go [hunk 1]
```diff
@@ -50,41 +50,19 @@ func (u *Manager) Close() error {
 }
 
 func (u *Manager) CreateOperatorUser(pass string) error {
-	tx, err := u.db.Begin()
+	_, err := u.db.Exec("CREATE USER IF NOT EXISTS 'operator'@'%' IDENTIFIED BY ?", pass)
 	if err != nil {
-		return errors.Wrap(err, "begin transaction")
+		return errors.Errorf("create operator user: %v", err)
```
Comment (egegunes) [inline pkg/pxc/users/users.go:55]: @theTibi please use `errors.Wrap` here and other places

**Review by egegunes — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/big-data.log) |
| cross-site | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/cross-site.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/scheduled-backup.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/storage.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/recreate.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/haproxy.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/upgrade-haproxy.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/scaling-proxysql.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/self-healing.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/self-healing-chaos.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/init-deploy.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/restore-to-encrypted-cluster.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/security-context.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/self-healing-advanced.log) |
| self-healing-advanced-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/self-healing-advanced-chaos.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/upgrade-proxysql.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/limits.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/demand-backup.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/monitoring-2-0.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/operator-self-healing-chaos.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/smart-update.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/affinity.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/one-pod.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/pitr.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/auto-tuning.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/users.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/upgrade-consistency.log) |
| demand-backup-encrypted-with-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/demand-backup-encrypted-with-tls.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/tls-issue-self.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/tls-issue-cert-manager.log) |
| tls-issue-cert-manager-ref | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/tls-issue-cert-manager-ref.log) |
| validation-hook | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/ac88f98d/validation-hook.log) |
| We run 35 out of 35|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1115/commits/ac88f98db8750192818e3357af28462e2601b19f
image: `perconalab/percona-xtradb-cluster-operator:PR-1115-ac88f98d`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/recreate.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/upgrade-haproxy.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/big-data.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/haproxy.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/scaling.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/cross-site.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/storage.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/scheduled-backup.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/scaling-proxysql.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/self-healing.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/self-healing-chaos.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/init-deploy.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/restore-to-encrypted-cluster.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/security-context.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/self-healing-advanced.log) |
| self-healing-advanced-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/self-healing-advanced-chaos.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/upgrade-proxysql.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/limits.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/demand-backup.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/monitoring-2-0.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/operator-self-healing-chaos.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/smart-update.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/affinity.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/one-pod.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/auto-tuning.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/users.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/upgrade-consistency.log) |
| pitr | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/pitr.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/tls-issue-self.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/tls-issue-cert-manager.log) |
| tls-issue-cert-manager-ref | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/tls-issue-cert-manager-ref.log) |
| validation-hook | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/validation-hook.log) |
| We run 34 out of 35|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1115/commits/b17b656bc61c7e7b3ab6f0310c6b6d470abc28ee
image: `perconalab/percona-xtradb-cluster-operator:PR-1115-b17b656b`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/recreate.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/cross-site.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/upgrade-haproxy.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/scaling.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/big-data.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/storage.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/scheduled-backup.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/restore-to-encrypted-cluster.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/haproxy.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/self-healing.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/scaling-proxysql.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/init-deploy.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/security-context.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/self-healing-chaos.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/pitr.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/limits.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/self-healing-advanced.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/monitoring-2-0.log) |
| self-healing-advanced-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/self-healing-advanced-chaos.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/affinity.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/operator-self-healing.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/one-pod.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/operator-self-healing-chaos.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/auto-tuning.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/users.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/tls-issue-self.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/tls-issue-cert-manager.log) |
| tls-issue-cert-manager-ref | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/tls-issue-cert-manager-ref.log) |
| validation-hook | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/validation-hook.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/upgrade-proxysql.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/smart-update.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/upgrade-consistency.log) |
| demand-backup-encrypted-with-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1115/b17b656b/demand-backup-encrypted-with-tls.log) |
| We run 35 out of 35|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1115/commits/b17b656bc61c7e7b3ab6f0310c6b6d470abc28ee
image: `perconalab/percona-xtradb-cluster-operator:PR-1115-b17b656b`

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**